SUMMARY.md

```diff
@@ -2,81 +2,107 @@
-  * [How do I enable site-wide full text search?](tutorials/howtos/search.md)
-  * [Building combined menus and custom buttons with the admin bar](tutorials/howtos/admin-bar.md)
-  * [Accessing the children of pages in a join](tutorials/howtos/children-and-joins.md)
-  * [Redirecting the user after they log in](tutorials/howtos/redirecting-after-login.md)
-  * [Facebook open graph tags in Apostrophe](tutorials/howtos/facebook.md)
-  * ["Next" and "previous" links for pieces](tutorials/howtos/next-and-previous-pieces.md)
-  * [Adding fields to all pieces](tutorials/howtos/adding-fields-to-all-pieces.md)
-  * [Lean frontend assets: Apostrophe without jQuery](tutorials/howtos/lean-frontend-assets.md)
-  * [Sending user notifications](tutorials/howtos/sending-user-notifications.md)
-  * [Adding new batch operations for pieces](tutorials/howtos/adding-new-batch-operations.md)
-  * [Extending the pieces editor modal](tutorials/howtos/extending-the-pieces-editor-modal.md)
-  * [Responsive images](tutorials/howtos/responsive-images.md)
-  * [Configuring CKEditor in Apostrophe](tutorials/howtos/ckeditor.md)
-  * [Storing images and files in Amazon S3](tutorials/howtos/storing-images-and-files-in-amazon-s3.md)
-  * [Storing images and files in Azure Blob Storage](tutorials/howtos/storing-images-and-files-in-azure-blob-storage.md)
-  * [Storing images and files in Google Cloud Storage](tutorials/howtos/storing-images-and-files-in-google-cloud-storage.md)
-  * [Migrating from Apostrophe 0.5](tutorials/howtos/migration.md)
-  * [Deploying Apostrophe to a Linode linux server quickly](tutorials/howtos/linode.md)
-  * [Deploying Apostrophe in the Cloud with Heroku](tutorials/howtos/deploying-apostrophe-in-the-cloud-with-heroku.md)
-  * [Deploying Apostrophe with AWS and Elastic Beanstalk](tutorials/howtos/deploying-apostrophe-in-the-cloud-with-aws.md)
-  * [Running Apostrophe on multiple cores and/or servers](tutorials/howtos/multicore.md)
-  * [Running Apostrophe on Windows](tutorials/howtos/windows.md)
-  * [Building Docker images for Apostrophe projects](tutorials/howtos/docker.md)
-  * [Using MongoDB replica sets with Apostrophe](tutorials/howtos/replica-set.md)
-  * [Storing sessions in Redis and other session stores](tutorials/howtos/storing-sessions-in-redis.md)
-  * [Logging in via Google, Twitter, Facebook and other sites](tutorials/howtos/google-twitter-and-facebook-login.md)
-  * [How can I remove pieces from an index listing?](tutorials/howtos/trim-index.md)
-  * [How can I disable a pieces index page?](tutorials/howtos/hide-index-page.md)
-  * [What to do when you are locked out of Apostrophe](tutorials/howtos/what-to-do-when-you-are-locked-out-of-apostrophe.md) 
-  * [Internationalization and localization](tutorials/howtos/apostrophe-i18n-config.md)
-  * [Nested module folders](tutorials/howtos/nested-module-folders.md)
+## Core Concepts
+* [Technical overview: 30,000 feet](tutorials/core-concepts/technical-overview.md)
```

Comment:
URLs are changing, so you will need to prepare a redirect mapping for gitbook so we don't lose all our SEO:

https://docs.gitbook.com/integrations/github/content-configuration

Reply:
Good call. I'll get working on that.